ubuntu_15.04.sh

```diff
@@ -50,6 +53,9 @@ cd PyTAPS-1.4/
 python setup.py --iMesh-path=$HOME/opt/moab --without-iRel --without-iGeom install --user
 cd ..
 # Install PyNE
+if [ -d pyne ] ; then
+    rm -rf pyne
+fi 
```

Comment:
Perhaps a warning should be given before deleting the pyne folder (or any folder). While it is assumed to be a clean machine, it is best to not accidentally delete a folder that may contain other things that just happens to have the same name. Perhaps having an interactive warning message to ask them if they want to proceed?